docs/general-settings.md

```diff
@@ -13,8 +13,8 @@ settings: {
   }
 }
 ````
-**hideNavigation** disables Luigi's default out-of-the-box navigation when set to `true`. Subsequently, neither the left nor top navigation are visible in your application. You are then free to provide your own navigation UI. By default, the navigation is enabled. If you want to keep it, set the value to `false` or leave this option unconfigured.
-**backdropDisabled** When set to `true`, prevents the backdrop layer from covering the top and left navigation when showing modal windows. By default, the backdrop is enabled and covers the top and left navigation when showing modal windows. To keep the default configuration, set the value to `false` or leave this option unconfigured.
-**header.logo** defines the top left navigation logo. It has a fixed height of 28px.
-**header.title** defines the top left navigation title.
-**header.favicon** defines the favicon. Requires a standard favicon file with the `.ico` extension, and 16x16px or 32x32px dimensions.
+* **hideNavigation** disables Luigi's default out-of-the-box navigation when set to `true`. As a result, neither the left nor top navigation are visible in your application, and you can implement your own navigation UI. By default, the navigation is enabled.
+* **backdropDisabled** When set to `true`, it prevents the backdrop layer from covering the top and left navigation when showing modal windows. By default, the backdrop is enabled.
```

Comment:
Something is needed here between the parameter's name and When. Paraphrasing should do the thing.

Reply:
👍